src/lsst/cmservice/__init__.py

```diff
@@ -1,14 +1,5 @@
-"""Campaign mangagement tooling"""
+"""Campaign management tooling"""
 
 __all__ = ["__version__"]
 
-from importlib.metadata import PackageNotFoundError, version
-
-__version__: str
-"""The application version string (PEP 440 / SemVer compatible)."""
-
-try:
-    __version__ = version("lsst-cm-service")
-except PackageNotFoundError:  # pragma: no cover
-    # package is not installed
-    __version__ = "0.0.0"
+__version__ = "0.1.4"
```

Comment:
Is this the number of PR's since @tcjennings became project technical lead?

Reply:
Not exactly, that's just the most recent existing tag in the repo.